cockroachcloud/export-logs.md

```diff
@@ -249,7 +249,7 @@ Perform the following steps to enable log export from your {{ site.data.products
 	1. Click on the name of your cluster.
 	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`.
 
-1. Determine the service account to use for exporting logs from your {{ site.data.products.dedicated }} cluster. This log export service account requires specific naming syntax to be recognized by the log export feature. This command uses the third-party JSON parsing tool [`jq`](https://stedolan.github.io/jq/download/) to isolate just the needed `id` (GCP cluster ID) and `account_id` (GCP account ID) fields, and combines them for you in the required form:
+1. Determine the {{ site.data.products.db }} service account to use for exporting logs from your {{ site.data.products.dedicated }} cluster. This log export service account requires specific naming syntax to be recognized by the log export feature. This command uses the third-party JSON parsing tool [`jq`](https://stedolan.github.io/jq/download/) to isolate just the needed `id` (GCP cluster ID) and `account_id` (GCP account ID) fields, and combines them for you in the required form:
```

Comment:
This is not correct. The service account here is the **GCP** service account not the CockroachCloud one. Sorry for causing more confusion 😓 .

Here's what I think we should do: let's not use the phrase "service account" at all here and say "principal" instead. So something like "Determine the GCP Principal to grant permissions to from your account". The user needs the name of the thing to allow to write to their GCP logs. That thing happens to be a service account also but we can say Principal here because in step 5c we say to grant access to this Principal.

Furthermore I think under "The resulting log export service account should resemble the following example:" we should have some sort of warning or note that says: "This principal refers to a resource that is owned by Cockroach Labs and is created automatically along with your cluster" to clarify that they should **not** create that service account when they see that string.

Let me know what you think. Thanks for iterating on this.

Reply:
Thanks @dhartunian, your suggested copy is very helpful! I might have succeeded in re-confusing myself, so please let me know if my changes get it right, or if I need to make more tweaks -- apologies if so.

Thanks so much for the help!